src/core/api/get_player_state.ts

```diff
@@ -74,6 +75,10 @@ export default function getLoadedContentState(
       return PLAYER_STATES.ENDED;
     }
 
+    if (stalledStatus.reason === "internal-seek") {
```

Comment:
Could be merged in the if at the top (`if (stalledStatus !== null && stalledStatus.reason !== "internal-seek")`).

On this subject, maybe it could be even more readable as the opposite (`if (stalledStatus === null || stalledStatus.reason === "internal-seek"`) because we would return earlier

Reply:
Also, now that I think of it. Wouldn't we want to still be sending "BUFFERING" states if we are "BUFFERING"?

Isn't it just "SEEKING" that we want to avoid? If we are clearly stalled, it makes no sense to have "PLAYING" as the player's state.

Reply:
Done